examples/vite.config.js

```diff
@@ -0,0 +1,13 @@
+import react from '@vitejs/plugin-react'
+import { resolve } from 'path'
+import { defineConfig } from 'vite'
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [react()],
+  resolve: {
+    alias: {
+      '@seamapi/third-party-assets': resolve(__dirname, '..'),
```

Comment:
Let's use tsConfigPaths: https://github.com/seamapi/react/blob/main/examples/vite.config.ts#L22

This can be vite.config.ts as well.

Non-blocking, can be handled in separate PR.